en/releases/centreon-core.md

```diff
@@ -19,8 +19,40 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.04.10
+
+#### Enhancements
+
+- [Configuration] Add the 'instance_heartbeat_interval' parameter in Engine configuration
+- [Configuration] Add the 'postpone_notification_to_timeperiod' parameter in Engine configuration
+- [Configuration] Improve access to the list of pollers
+- [Core] Performance improvements for partitioning
+- [Core] Update PHP 7.3 compatibility
+- [Core] Use Gorgone to dispatch downtimes locally
+- [Resources Status] Display of comments in the host details page
+- [Top counters] Displayed values for services don't consider host acknowledgments
+
+#### Bugfixes
+
+- [CLAPI] No control on dependencies relations
+- [Configuration] Non-admin users can't create host/service
+- [Remote Server] Import task failed when a Poller is linked to additional RS
```

Comment:
this fix must not be mentionned in the RN for this version (nor in 20.10.4) because the changes made by QGA are not sufficient to fix the issue. QA only checked that there is no regression for the usual import task (Central + RS or Central + RS + Poller). There is another ticket linked to MON-6541 not started yet. I had to set the status of MON-6541 to Resolved for us to be able to release the version in JIRA.